nbhttp/websocket/conn.go

```diff
@@ -142,6 +142,17 @@ func (c *Conn) OnMessage(h func(*Conn, MessageType, []byte)) {
 	}
 }
 
+func (c *Conn) OnDataFrame(h func(*Conn, MessageType, bool, []byte)) {
```

Comment:
Named OnFrameData seems a little better.

Reply:
My reason for using OnDataFrame is that the only frames that will be sent to this function are Text and Binary. Ping and Close messages can also have Frame Data, but they will will not passed to this function.

Reply:
oh yes, I got it.